test/integration/clinicians/clinician-modal.js

```diff
@@ -0,0 +1,217 @@
+import _ from 'underscore';
+import { getError } from 'helpers/json-api';
+
+const stateColors = Cypress.env('stateColors');
+
+const groups = [
+  {
+    id: '1',
+    name: 'Group One',
+  },
+  {
+    id: '2',
+    name: 'Group Two',
+  },
+];
+
+context('clinicians modal', function() {
+  specify('add clinician', function() {
```

Comment:
this was adapted from the similar tests when we were adding them from the sidebar